handlers/configuration.go

```diff
@@ -0,0 +1,5 @@
+package handlers
+
+const (
+	CREDENTIAL_PATH = "./credentials.json" // AWS credential
```

Comment:
can this be maybe an optional environment variable that points to a file?

Reply:
I prefer that way only since all our micro services are using the same approach.

Reply:
Replace it with Lookup function